docs/tutorial/gatsby-image-tutorial

```diff
@@ -0,0 +1,275 @@
+
+### Yaml and Images
+
+In order to reference your images in yaml make sure that the relative path is accurate. The path to the image should be relative to the location of the `.yaml` file pointing to it. And all of these files need to be in a directory visible to the `gatsby-source-filesystem` plugin.
+
+The inside of the yaml file would look something like this.
+
+```
+- image: speaking/kcdc.jpg
+```
+
+Now, you can create the query. Similar to the single use example above, you can use `gatsby-image` features inside the query. When the query runs, the relative path will point to the location of the image file and the resulting query processes the file as an image for display.
+
+```jsx
+{
+    allSpeakingYaml {
+        edges {
+            node {
+                image {
+                    childImageSharp {
+                        fluid {
+                            ...GatsbyImageSharpFluid
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+Since the images are stored as part of an array, they can be accessed using the map function in JSX. As with the single image example, the actual processed image is at the `...GatsbyImageSharpFluid` level.
+
+```jsx
+<Img
+  className="selfie"
+  fluid={node.image.childImageSharp.fluid}
+  alt={node.conference}
+/>
+```
+
+### Static Query
+
+If your query is part of a reusable component you may want to use a Static Query hook. The code necessary to do this is almost the same as the single image use case above.
+
+```jsx
+export default () => {
+  const data = useStaticQuery(graphql`
+    query {
+      file(relativePath: { eq: "headers/default.jpg" }) {
+        childImageSharp {
+          fixed(width: 125, height: 125) {
+            ...GatsbyImageSharpFixed
+          }
+        }
+      }
+    }
+  `)
+
+  return <Img fixed={data.file.childImageSharp.fixed} />
+}
+```
+
+Instead of a query constant and data that references the result, you can use a static query hook directly in the JXS code and then reference it. Note that the query language didn’t change and neither did the `Img` tag syntax, the only change was the location of the query and the usage of the `useStaticQuery` function to wrap it.
+
+### Multiple Queries and Aliasing
+
+The last use case you may come across is how to handle a situation where you have multiple queries in the same file/page.
+
+This example is attempting to query for all the data in `speaking.yaml` and the direct file query in our first example. In order to do this you want to use aliasing.
```

Comment:
```suggestion
This example is attempting to query for all the data in `speaking.yaml` and the direct file query in our first example. In order to do this you want to use aliasing in GraphQL.
```
Adding a qualifier here in case people want to go find an API for aliasing and they don't know where to look!